### ISSUE TITLE
Version 1.8 don't work with Symbols

### ISSUE THREAD

**Issue by funk4d** [kenmoore/sketch-relabel-button#44]
Plugin version 1.8
Sketch version 53

**Comment by kenmoore** [comment]
So this worked with symbols in version 1.7?

**Comment by funk4d** [comment]
Yes

**Comment by kenmoore** [comment]
@bomberstudios would you mind having a look into this new issue?  Thanks

**Comment by denis-matveev** [comment]
Yep same here. Stopped working after updating to 1.8

**Comment by alexeiboronnikov** [comment]
Same.Up.

**Comment by bomberstudios** [comment]
I'll see what I can do!

**Comment by durmuskavcioglu** [comment]
@kenmoore  I have problem too with version 1.8. 

I'm using Sketch v53.1 and this plugin version. Also my friend have Sketch v53.1 and Relabel Button 1.8, but it's working with his computer. It's doesn't working my computer. I couldn't find of the last version 1.7.  I don't understand this situation. Please solve this problem. Is there any solition?

**Comment by funk4d** [comment]
@durmuskavcioglu enjoy!
[Relabel Button 1.7.sketchplugin.zip](https://github.com/kenmoore/sketch-relabel-button/files/2875080/Relabel.Button.1.7.sketchplugin.zip)

**Comment by durmuskavcioglu** [comment]
@funk4d Thank you. It's working now. But v1.8 doesn't working in Sketch 53.1. I think it will be good in short time. 

Also, another bug in the Sketch 53.1; " When I press the Command + J, it doesn't automatic select the placeholder when the window is opened."  But before the Sketch 53 versions, it was working with automatic select placeholder.

**Comment by funk4d** [comment]
@durmuskavcioglu 

> When I press the Command + J, it doesn't automatic select the placeholder when the window is opened.

Yes, I know, BTW this is temporary solution. We are waiting for a fix by the developer.
